### ISSUE TITLE
Module parse failed: 'import' and 'export' may appear only with 'sourceType: module'

### ISSUE THREAD

**Issue by kpeters-cbsi** [okta/okta-auth-js#1017]
When I updated a project that had previously been using `@okta/okta-angular@3.2.2` to use `@okta/okta-angular@4.1.0` and `@okta/okta-auth-js@5.9.0` and tried to run `ng serve`, I got the below. However, the behavior did not occur with `@okta/okta-auth-js@5.6.0`.  Using Angular 13 and `@angular-devkit/build-angular@13.0.4` (which installed `webpack@5.64.1`)

```
./node_modules/@okta/okta-auth-js/esm/index.js:12:0 - Error: Module parse failed: 'import' and 'export' may appear only with 'sourceType: module' (12:0)
File was processed with these loaders:
 * ./node_modules/@angular-devkit/build-angular/src/babel/webpack-loader.js
 * ./node_modules/@angular-devkit/build-angular/node_modules/@ngtools/webpack/src/ivy/index.js
 * ./node_modules/@angular-devkit/build-angular/node_modules/source-map-loader/dist/cjs.js
You may need an additional loader to handle the result of these loaders.
|  * See the License for the specific language governing permissions and limitations under the License.
|  */
> import _classCallCheck from '@babel/runtime/helpers/classCallCheck';
| import _inherits from '@babel/runtime/helpers/inherits';
| import _possibleConstructorReturn from '@babel/runtime/helpers/possibleConstructorReturn';
```

**Comment by shuowu** [comment]
@kpeters-cbsi Thanks for reporting the issue! We actually also noticed this issue after 5.9.0 release, and we are working on a patch now.

Internal Ref: OKTA-451360

**Comment by arahate** [comment]
Facing same issue.

**Comment by kpeters-cbsi** [comment]
I got around it by downgrading to 5.8.0

**Comment by kpeters-cbsi** [comment]
But it looks like there's a fix pending code review

**Comment by shuowu** [comment]
Patch for this issue is released in okta-auth-js v5.9.1. Closing this issue, feel free to re-open if the problem still exists.
